Token.js

```diff
@@ -166,6 +166,22 @@ class Token {
 
 			this.options.hp = old.find(".hp").val();
 			this.options.max_hp = old.find(".max_hp").val();
+
+			// HEALTH AURA
+			if (this.options.max_hp > 0) {
+
+				if (this.options.max_hp > 0 && parseInt(this.options.hp) === 0) {

```

Comment:
`this.options.max_hp > 0` is already determined on line 171.
I think we can remove it from line 173.  :)